muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/connect-field.tsx

```diff
@@ -225,7 +225,7 @@ export default class ConnectField extends React.Component<ConnectFieldProps, Con
     this.checkAnswers();
   }
   //swapping the fields
-  swapField(fielda: FieldType, fieldb: FieldType){
+  swapField(triggerChange: boolean, fielda: FieldType, fieldb: FieldType){
```

Comment:
fieldA, fieldB?

Reply:
Yes that's what they mean, it would be nicer to name them fieldA and fieldB in camel case but they were originall named like that, I'd have to otherwise rename everywhere they were used.

Reply:
I don't see the problem in it, but fair enough. Not a big issue.